create_mod_core.py

```diff
@@ -27,35 +35,55 @@ def clean_data(initial_dataset: pd.DataFrame):
                                             skippable=True)
 
 
-model_training_cfg = Config.configure_data_node(id="model_output",
-                                                storage_type= 'pickle',
-                                             scope=Scope.GLOBAL)                                                  
-
-def retrained_model(cleaned_dataset: pd.DataFrame):
-    print("Model Retraining")
-    model= Prophet()
-    model.fit(cleaned_dataset)
-
-    with open('aapl_model.pkl', 'wb') as f:
-        pickle.dump(model, f)
+model_training_cfg = Config.configure_data_node(id="model_output")                                                  
 
-    return model
+predictions_cfg = Config.configure_data_node(id="predictions")                                                  
 
 model_training_task_cfg = Config.configure_task(id="model_retraining_task",
                                                   function=retrained_model,
-                                                  input=[cleaned_dataset_cfg],
+                                                  input=cleaned_dataset_cfg,
                                                   output=model_training_cfg)
 
+predict_task_cfg = Config.configure_task(id="predict_task",
+                                                  function=predict,
+                                                  input=model_training_cfg,
+                                                  output=predictions_cfg)
+
+# Create our scenario configuration from our tasks
+scenario_cfg = Config.configure_scenario_from_tasks(id="stock",
+                                                    task_configs=[clean_data_task_cfg,
+                                                    model_training_task_cfg,
+                                                    predict_task_cfg])
+
+
+"""
```

Comment:
Do you want to keep it?

Reply:
Yes, it was in his code. I think he uses it for the first steps